developer_manual/design/foundations.rst

```diff
@@ -129,6 +133,8 @@ While interface elements like buttons are colored differently depending on their
 Typography
 ----------
 
+`Penpot Typographie <https://design.penpot.app/#/view/db3839da-807b-8052-8002-576401e9a375?page-id=3f784c86-6c27-80c6-8002-6ab128f3ffe2&section=interactions&index=1&share-id=11fde340-21f4-802e-8002-8ebec1b418b7>`_
```

Comment:
Link change cause of permissions and typo:
```suggestion
`Penpot typography <https://design.penpot.app/#/view/db3839da-807b-8052-8002-576401e9a375?page-id=3f784c86-6c27-80c6-8002-6ab128f3ffe2&section=interactions&index=1&share-id=11fde340-21f4-802e-8002-8d8d305e7ab5>`_
```

Reply:
done